alt_exit_test.go

```diff
@@ -45,6 +45,8 @@ func TestHandler(t *testing.T) {
 	if string(data) != arg {
 		t.Fatalf("bad data. Expected %q, got %q", data, arg)
 	}
+	fmt.Printf(tempDir)
```

Comment:
printf-style function with dynamic first argument and no further arguments should use print-style function instead